index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework</title>
+  </head>
+  <body>
+    <main>
+      <header>
+        <section>
+          <nav>
+            <div>
+              <img src="icons/Logo.svg" alt="logo" />
+            </div>
+          </nav>
+        </section>
+        <section>
+          <ul>
+            <li>Product</li>
+            <li>Customers</li>
+            <li>Pricing</li>
+            <li>Resources</li>
+          </ul>
+        </section>
+        <section>
+          <button type="button" class="btn btn-secondary">Sign in</button>
+          <div>
+            <button type="button" class="btn btn-warning">Sign up</button>
+          </div>
+        </section>
```

Comment:
- `header` закінчується тут
- усі теги section всередині `header` зайві. Тег section описує якийсь окремий блок сайту, а не просто розмежовує меленькі частинки і скоріш за все тег `section` містить заголовок
- лого майже завжди це посилання на головну сторінку

```suggestion
      <header>
        <nav>
          <a href="/">
            <img src="icons/Logo.svg" alt="logo" />
          </a>
        </nav>
        <ul>
          <li>Product</li>
          <li>Customers</li>
          <li>Pricing</li>
          <li>Resources</li>
        </ul>
        <div>
            <button type="button" class="btn btn-secondary">Sign in</button>
            <button type="button" class="btn btn-warning">Sign up</button>
          </div>
      </header>
```